griffon/services/core_queries.py

```diff
@@ -281,11 +265,11 @@ def __init__(self, params: dict) -> None:
             self.community_session = CommunityComponentService.create_session()
 
     def execute(self, status=None) -> List[Dict[str, Any]]:
+        max_limit = 5000
         status.update("griffoning: searching component-registry.")
-        item_limit = 50
         results = []
         params = {
-            "include_fields": "link,purl,type,name,related_url,namespace,software_build,nvr,release,version,arch,product_streams.product_versions,product_streams.name,product_streams.ofuri,product_streams.active,product_streams.exclude_components,sources.nvr,sources.purl,sources.name,sources.namespace,sources.download_url,sources.related_url,upstreams.nvr,upstreams.purl,upstreams.namespace,upstreams.name,upstreams.download_url,upstreams.related_url",  # noqa
+            "include_fields": "link,purl,type,name,related_url,namespace,software_build,nvr,release,version,arch,product_streams.product_versions,product_streams.name,product_streams.ofuri,product_streams.active,product_streams.exclude_components",  # noqa
```

Comment:
Note: I will try to soon support in bindings for python native list as a way to provide comma separated list as a query parameter so this can be written nicer.